.travis.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
@roidelapluie Shouldn't this file be in a separate commit? This is part of the Travis support for test running, not part of the recursive ACL changes, right?